### PR TITLE
Update required GITHUB_TOKEN permissions

### PR DESCRIPTION
**Pull request by pellared**
`contents: read` should be enough when signatures are stored in a remote repository.

Tested here: https://github.com/pellared/test-cla/pull/1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,10 +27,10 @@ on:
 
 # explicitly configure permissions, in case your GITHUB_TOKEN workflow permissions are set to read-only in repository settings
 permissions:
-  actions: write
-  contents: write
+  actions: read
```
Comment (pellared) [inline README.md:30]: Maybe there is some palce where `write` is needed?

File: README.md [hunk 2]
```diff
@@ -27,10 +27,10 @@ on:
...
+  contents: write # this can be 'read' if the signatures are in remote repository
   pull-requests: write
-  statuses: write
+  statuses: read
```
Comment (pellared) [inline README.md:33]: Maybe there is some palce where `write` is needed?

File: README.md (hunk 1 above)
Comment (pellared) [reply README.md:30]: AFAIK this is needed when someone adds `recheck` comment

File: README.md (hunk 2 above)
Comment (pellared) [reply README.md:33]: This is needed when the workflow wants to update the status check.

**Comment by pellared** [comment]
@ibakshay Can you please take a look?

**Comment by ibakshay** [comment]
@pellared, 
I am really sorry. I will have a look in the next coming days and merge this PR :) thanks so much for the contribution! I highly appreciate it..

**Review by ibakshay — APPROVED** [review]
Thank you for the contribution!

**Pull request merged**